src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/ArtworkFilterRefetch.tsx

```diff
@@ -81,6 +81,8 @@ export const ArtworkFilterRefetchContainer = createRefetchContainer(
           major_periods: { type: "[String]" }
           partner_id: { type: "ID" }
           for_sale: { type: "Boolean" }
+          at_auction: { type: "Boolean" }
```

Comment:
this just propagates the query vars around, from the query renderer -> fragments.